bears/generate_package.py

```diff
@@ -87,15 +98,48 @@ def create_upload_parser():
     :return: Parser arguments.
     """
     parser = argparse.ArgumentParser(
-        description='Generates PyPi packages from bears.')
+        description='Generates PyPI packages from bears.')
     parser.add_argument('-r', '--register',
-                        help='Register the packages on PyPi',
+                        help='Register the packages on PyPI',
                         action='store_true')
-    parser.add_argument('-u', '--upload', help='Upload the packages on PyPi',
+    parser.add_argument('-u', '--upload', help='Upload the packages on PyPI',
```

Comment:
Would have been nice to have these PyPi ->PyPI in another commit. It's distracting when reading the diff